09-model-mlr.Rmd

```diff
@@ -646,9 +646,27 @@ In this example, we have arrived at the same model that we identified from backw
 
 Backward elimination begins with the model having the largest number of predictors and eliminates variables one-by-one until we are satisfied that all remaining variables are important to the model.
 Forward selection starts with no variables included in the model, then it adds in variables according to their importance until no other important variables are found.
-
-Backward elimination and forward selection sometimes arrive at different final models.
-If trying both techniques and this happens, it's common to choose the model with the larger adjusted $R^2.$
+Notice that, for both methods, we have always chosen to retain the model with the largest adjusted $R^2$ value, even if the difference is less than half a percent (e.g., 0.2597 versus 0.2598).
+One could argue that the difference between these two models is negligible, as they both explain nearly the same amount of variability in the `interest_rate`.
+These negligible differences are an important aspect to model selection.
+It is highly advised that *before* you begin the model selection process, you decide what a "meaningful" difference in adjusted $R^2$ is for the context of your data.
+Maybe this difference is 1% or maybe it is 5%.
+This "threshold" is what you will then use to decide if one model is "better" than another model.
+Using meaningful thresholds in model selection requires more critical thinking about what the adjusted $R^2$ values mean!
+
+Additionally, you should know that backward elimination and forward selection sometimes arrive at different final models.
+This is because the decision for whether to include a variable or not depends on what other variables are already in the model.
```

Comment:
```suggestion
This is because the decision for whether to include a given variable or not depends on the other variables are already in the model.
```